### PR TITLE
Rib 62 implement interest page

### PR DESCRIPTION
**Pull request by marthubner**
## RIB-62: Implement interest page
- Implementing styles for interest/page.html
- Adding necessary backend logic in PlaceService and DTO for sending the data to template
- Enhancing rating.js and fixing rating number on frontend to be in line with number stars 

### Left to do in interest/page.html
- Implement search bar
- Implement top bar menu with like/dislike button, edit interest, request rating access button - all based on users role
- Implement category filtering

### CODE REVIEW

**Review by vdrfg — APPROVED** [review]
Damn

**Review by BenediktVitek — APPROVED** [review]

File: src/main/java/it/rate/webapp/services/PlaceService.java [hunk 1]
```diff
@@ -70,16 +71,58 @@ public Place getReferenceById(Long placeId) {
...
+    place
+        .getInterest()
+        .getCriteria()
+        .forEach(
+            criterion -> criteriaAvgRatingDTOs.add(getCriterionAvgRatingDTO(criterion, place)));
+
     return new CriteriaOfPlaceDTO(criteriaAvgRatingDTOs);
   }
+
+  public List<PlaceInfoDTO> getPlaceInfoDTOS(Interest interest) {
```
Comment (MatejFrnka) [inline src/main/java/it/rate/webapp/services/PlaceService.java:85]: unit test this method

**Review by MatejFrnka — COMMENTED** [review]
add at least 2 unit tests for the getPlaceInfoDTOS. Everything else looks good

**Comment by dondowoldo** [comment]
Looks amazing on PC, but on mobile the elements don't align properly. Not sure if we need to address this before merging but it definitely needs a little tweak in css.

**Review by dondowoldo — APPROVED** [review]

**Pull request merged**